frontend/src/lib/Quiver.ts

```diff
@@ -37,9 +37,14 @@ interface TableCell {
   content: string
 }
 
-export interface Index {
+export interface Data {
   data: any[][]
-  type: IndexType
+  type: string[]
```

Comment:
Because understanding the data shape is so important for this arrow stuff, it'd be great to have type and member docstrings that explain this more. I'm inventing this example because I don't actually know what the "Data" type means, but something like:

```
/** Contains data and type information for a single index. */
export interface Data {
  /** The data entries that make up the index. Each entry is an array of data containing blah blah blah. */
  data: any[][]
  /** Type typenames of each foo in the bar. */
  type: string[]
}
```

Reply:
Done